### DISCUSSION TITLE
How to set the frame interval for a WebP animation?

### DISCUSSION THREAD

**Discussion by xuanhen2013** [ImageMagick/ImageMagick#6815] [Help] [answered]
### ImageMagick version

7.1.1-21

### Operating system, version and so on

mac

### Description

`convert image-*.png -delay 33.33 out4.webp`
The -delay number doesn't work.

**Comment by snibgo** [comment]
Try the delay *before* reading the input files.

**Comment by snibgo** [comment, marked as answer]
I also suggest you use `magick` instead of `convert`.

A quick test show that both of these commands work:
```
magick -delay 100 rose: wizard: -resize "200x200^!" x.webp

magick rose: wizard: -resize "200x200^!" -set delay 100 x.webp
```
